modules/Db2/templates/db2-namespace.template.yaml

```diff
@@ -0,0 +1,14 @@
+###############################################################################
```

Comment:
This should be named db2-namespace.yaml.tmpl so that it won't ever be interpreted as a yaml until the necessary variables have been replaced.

Reply:
I restructured and removed it.